n_motors: 6               # [-]
mass: 2.0                 # [kg]
arm_length: 0.27          # [m]
body_height: 0.1          # [m]
motor_time_constant: 0.03 # [s]
g: 9.81                   # [ms^-2]
air_resistance_coeff: 0.3 # TODO

propulsion:

  # force [N] = force_constant * rpm^2
  force_constant: 0.00000012216

  # moment [Nm] = moment_constant * force [N]
  moment_constant: 0.07

  prop_radius: 0.11 # [m]

  # allocation motors -> moments

  # hexarotor X configuration
  allocation_matrix: [
    1, -1, -0.5, 0.5, 0.5,  -0.5,
    0, 0,  -0.5, 0.5, -0.5, 0.5,
    1, -1, 1,    -1,  1,    -1,
    1, 1,  1,    1,   1,    1
  ]

rpm:
  min: 906   # [revolutions/minute]
  max: 9068  # [revolutions/minute]
